source/docs/software/pathplanning/index.rst

```diff
@@ -1,7 +1,7 @@
 Path Planning
 =============
 
-Path Planning is the process of developing and "planning" paths that your robot can autonomously follow. These paths use the WPILib Trajectory Framework for generation and Ramsete for following. This section highlights the process of characterizing your robot for system identification, developing with the trajectory tracking API and visualizing your paths with PathWeaver. Users may also want to read the :ref:`generic trajectory following documents <docs/software/advanced-controls/trajectories/index:Trajectory Generation and Following with WPILib>` for non-commandbased usage.
+ "Path Planning is the process of creating and following paths that your robot can autonomously follow. These paths use the WPILib trajectory APIs for generation and a :ref:`Ramsete Controller <docs/software/advanced-controls/trajectories/ramsete:Ramsete Controller>` for following. This section highlights the process of characterizing your robot for system identification, developing with the trajectory tracking API and visualizing your paths with PathWeaver. Users may also want to read the :ref:`generic trajectory following documents <docs/software/advanced-controls/trajectories/index:Trajectory Generation and Following with WPILib>` for non-commandbased usage.
```

Comment:
"developing with the trajectory tracking API and visualizing your paths with PathWeaver."

1) developing seems odd
2) Pathweaver generates too

I propose: trajectories can be created using either the API or generated in PathWeaver then imported.